test/Utils.test.js

```diff
@@ -66,3 +67,12 @@ describe('calculateDistance function', () => {
     expect(calculateDistance(coord1, coord2)).toBeCloseTo(559.1, 1);
   });
 });
+
+describe('getHumanDateFromEpochSeconds function', () => {
```

Comment:
Do we need a test for the `pad` case?

Reply:
It is tested in the second result. 5->05; 7->07 while 10 and 51 are unaffected